README.md

```diff
@@ -7,15 +7,38 @@
 
 
 
-Uxarray aims to address the geoscience community need for utilities that can handle 2D and 3D unstructured grid datasets.
-These utility functions were inspired by discussion on the [Xarray GitHub Repository](https://github.com/pydata/xarray/issues/4222).
-Uxarray will provide Xarray styled funtions to better read in and use unstructured grid datasets that follow UGRID conventions.
-This effort is a result of the collaboration between Project Raijin (NCAR and Pennsylvania State University)
-and SEATS Project (Argonne National Laboratory, UC Davis, and Lawrence Livermore National Laboratory).
-
-Uxarray is implemented in pure Python and does not explicitly contain or require any compiled code. This makes Uxarray more
-accessible to the general Python community. Any contributions to this repository in pure Python are welcome and documentation
-for contribution guidelines can be found when clicking `New Issue` under the `Issues` tab in the Uxarray repository.
+Uxarray aims to address the geoscience community need for tools that enable standard data analysis techniques to
+operate directly on unstructured grid data.  These utility functions were inspired by discussion on the
```

Comment:
The phrase starting with "These utility functions..." sounds a bit out of context in this first paragraph. I'd suggest:

- Moving that phrase to just before the part entitled "Intended Functionality for Grids"
- Rephrasing that sentence into something like: "The following intended functionality is inspired by the project plans of SEATS Project and Project Raijin as well as discussions on several community platforms such as [Xarray GitHub Repository](https://github.com/pydata/xarray/issues/4222)"
- Combining the first and second paragraphs into a single paragraph.

Reply:
How about this?

Uxarray aims to address the geoscience community need for tools that enable standard data analysis techniques to
operate directly on unstructured grid data.  Uxarray will provide Xarray styled functions to better read in and
use unstructured grid datasets that follow standard conventions, including UGRID, SCRIP, Exodus and shapefile
formats.  This effort is a result of the collaboration between Project Raijin (NCAR and Pennsylvania State University)
and the SEATS Project (Argonne National Laboratory, UC Davis, and Lawrence Livermore National Laboratory).

Uxarray is implemented in pure Python and does not explicitly contain or require any compiled code. This makes Uxarray more accessible to the general Python community. Any contributions to this repository in pure Python are welcome and documentation for contribution guidelines can be found when clicking `New Issue` under the `Issues` tab in the Uxarray repository.

The following intended functionality has been inspired by discussions with members of the scientific community,
within the SEATS Project and Project Raijin, and on several community platforms such as
[Xarray GitHub Repository](https://github.com/pydata/xarray/issues/4222).  The Uxarray team is receptive to additional functionality requests.

Reply:
It sounds great to me. I am approving it. Should we add something as follows to the end of first paragraph:

"The Uxarray teams welcome other community members to become part of this collaboration with any level of contribution."

Reply:
Added in final update